packages/flutter/lib/src/material/switch.dart

```diff
@@ -104,9 +104,18 @@ class Switch extends StatelessWidget {
       inactiveThumbDecoration: inactiveThumbDecoration,
       activeTrackColor: activeTrackColor,
       inactiveTrackColor: inactiveTrackColor,
+      configuration: createLocalImageConfiguration(context),
       onChanged: onChanged
     );
   }
+
+  @override
+  void debugFillDescription(List<String> description) {
+    super.debugFillDescription(description);
+    description.add('${value ? "checked" : "unchecked"}');
```

Comment:
fixed (though the render object will still say checked/unchecked since it's general)